lib/rdfc-processor.ts

```diff
@@ -83,13 +91,11 @@ export async function processor(
         const seen = new Set();
         while (el) {
             if (el.value) {
-                seen.add(el.value.id);
+                seen.add(el.value.id.value);
 
-                if (seen.size % 100 == 1) {
-                    logger.verbose(
-                        `Got member ${seen.size} with ${el.value.quads.length} quads`,
-                    );
-                }
+                logger.verbose(
```

Comment:
Logging only every 100th member looked quite fine to me to prevent logging too much. Any particular reason you want to move away from this?

Reply:
My reasoning was: This is mainly for debugging purposes and if I want to debug, I probably need to see what members are being emitted.

Reply:
Alright, logging every 100th member is back for `verbose` logs (https://github.com/rdf-connect/ldes-client/pull/94/commits/e49d643f56a2e606420bec5b9f0629e467be02ec). I removed logging every member in `debug` logs because getting every member is now possible on `stdout` and because it gets too messy to read the logs